### PR TITLE
Remove unused indexes

### PR DESCRIPTION
**Pull request by lucasvanhalst**
Run this command after deploying:

`gcloud datastore cleanup-indexes --project rogerthat-server index.yaml`

### CODE REVIEW

File: src/index.yaml [hunk 1]
```diff
@@ -225,11 +225,6 @@ indexes:
   properties:
   - name: number
 
-- kind: Product
-  properties:
-  - name: visible
-  - name: description
```
Comment (lucasvanhalst) [inline src/index.yaml]: description property no longer exists

File: src/index.yaml [hunk 2]
```diff
@@ -271,11 +266,6 @@ indexes:
   - name: timestamp
     direction: desc
 
-- kind: Question
-  properties:
-  - name: replied
-  - name: ask_time
```
Comment (lucasvanhalst) [inline src/index.yaml]: ask_time does not exist

File: src/index.yaml [hunk 3]
```diff
@@ -323,42 +313,10 @@ indexes:
   - name: app_ids
   - name: name
 
-- kind: RestaurantMessage
```
Comment (lucasvanhalst) [inline src/index.yaml]: RestaurantMessage does not exist

File: src/index.yaml [hunk 4]
```diff
@@ -417,11 +364,6 @@ indexes:
   properties:
   - name: order
 
-- kind: SandwichOptions
```
Comment (lucasvanhalst) [inline src/index.yaml]: `SandwichOptions` does not exist (should be without the 's' at the end

File: src/index.yaml [hunk 5]
```diff
@@ -672,14 +614,6 @@ indexes:
   - name: timestamp
     direction: desc
 
-- kind: SolutionLoyaltyVisit
```
Comment (lucasvanhalst) [inline src/index.yaml]: SolutionLoyaltyVisit does not exist

File: src/index.yaml [hunk 6]
```diff
@@ -395,17 +353,6 @@ indexes:
...
-- kind: RestaurantReservation
-  ancestor: yes
-  properties:
-  - name: shift_start
-  - name: date
-
-- kind: RestaurantReservation
-  ancestor: yes
-  properties:
-  - name: status
```
Comment (RubenMattan) [inline src/index.yaml]: Still in use

**Pull request merged**